test/tests/LayerStatesTest.coffee

```diff
@@ -3,6 +3,9 @@ assert = require "assert"
 
 initialStateName = "default"
 
+cleanState = (state) ->
+	return _.pickBy(state, (value, key) -> key isnt "name")
```

Comment:
This could have a better name then `cleanState`, maybe `removeKeysFromState(state,keys=["name"])`?